MBBSEmu/HostProcess/ExportedModules/Majorbbs.cs

```diff
@@ -3358,7 +3372,7 @@ private void rstrin()
             for (var i = inputPointer.Offset; i < inputPointer.Offset + (inputLength - 1); i++)
             {
                 if (Module.Memory.GetByte(inputPointer.Segment, i) == 0)
-                    Module.Memory.SetByte(inputPointer.Segment, i, 0x20);
+                    Module.Memory.SetByte(inputPointer.Segment, i, (byte)' ');
```

Comment:
ugh the cast is worse actually ;-) Maybe some nice constants would help, instead of 0x20 could have a constant called ASCII_SPACE or simply SPACE.

Reply:
If it's any consolation, the compiler is smart enough to convert it to a const at compile time 😉 

![image](https://user-images.githubusercontent.com/1139047/91555572-416a2000-e8ff-11ea-85f3-b32a53e31876.png)